config/traits.yml

```diff
@@ -340,98 +340,98 @@ traits:
       label: P content of spongy mesophyll cells
       allowed_values_min: 0
       allowed_values_max: 30
-    cell_thylakoid_N_per_total_N:
+    cell_thylakoid_N_per_total_cell_N:
       description: Percentage of N accounted for by thylakoid proteins
       type: numeric
       units: '%'
       label: Percentage of N accounted for by thylakoid proteins
       allowed_values_min: 5
       allowed_values_max: 30
-    chlorophyll_per_area:
+    leaf_chlorophyll_per_area:
       description: Sum of chlorophyll A and B per leaf area
       type: numeric
       units: umol/m2
       label: Sum of chlorophyll A and B per leaf area
       allowed_values_min: 10
       allowed_values_max: 2000
-    chlorophyll_A_per_area:
+    leaf_chlorophyll_A_per_area:
       description: Leaf chlorophyll A content per leaf area
       type: numeric
       units: umol/m2
       label: Leaf chlorophyll A content per leaf area
       allowed_values_min: 50
       allowed_values_max: 2000
-    chlorophyll_B_per_area:
+    leaf_chlorophyll_B_per_area:
       description: Leaf chlorophyll B content per leaf area
       type: numeric
       units: umol/m2
       label: Leaf chlorophyll B content per leaf area
       allowed_values_min: 10
       allowed_values_max: 1000
-    chlorophyll_per_dry_mass:
+    leaf_chlorophyll_per_dry_mass:
       description: Leaf chlorophyll content per unit leaf dry mass
       type: numeric
       units: mmol/kg
       label: Leaf chlorophyll content per unit leaf dry mass
       allowed_values_min: 0.1
       allowed_values_max: 15.0
-    chlorophyll_A_per_dry_mass:
+    leaf_chlorophyll_A_per_dry_mass:
       description: Leaf chlorophyll A content per unit leaf dry mass
       type: numeric
       units: mmol/kg
       label: Leaf chlorophyll A content per unit leaf dry mass
       allowed_values_min: 0.1
       allowed_values_max: 10.0
-    chlorophyll_B_per_dry_mass:
+    leaf_chlorophyll_B_per_dry_mass:
       description: Leaf chlorophyll B content per unit leaf dry mass
       type: numeric
       units: mmol/kg
       label: Leaf chlorophyll B content per unit leaf dry mass
       allowed_values_min: 0.01
       allowed_values_max: 10.0
-    chlorophyll_A_B_ratio:
+    leaf_chlorophyll_A_B_ratio:
       description: Ratio of leaf chlorophyll A to chlorophyll B
       type: numeric
       units: umol/umol
       label: Ratio of leaf chlorophyll A to chlorophyll B
       allowed_values_min: 0.1
       allowed_values_max: 6
-    ca:
+    atmospheric_CO2_concentration:
```

Comment:
This isn't really a trait but a context. No need to change now, but put on a list for later?

Reply:
sure - although we still can't really cope with continuous contexts so they probably do have to be input as traits.